src/CoreFx.Private.TestUtilities/src/System/RetryHelper.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    public static partial class RetryHelper
+    {
+        private static readonly Func<int, int> s_defaultBackoffFunc = i => Math.Min(i * 100, 60_000);
```

Comment:
Why did you chose 60 seconds to be the max delay in between retries? Just random?

Reply:
Arbitrary. But that's just the default. A caller can specific whatever function they want. We can change any and all of this as need arises.

Reply:
Ok, it just scared me looking at 60 seconds retry 😄 -- but sounds good, it is also hard to get to it, you would need to set max attempts to 60 and if a test does that, then we should definitely delete or fix that test.